src/engine/QueryExecutionTree.cpp

```diff
@@ -35,25 +35,23 @@ string QueryExecutionTree::asString(size_t indent) {
   for (size_t i = 0; i < indent; ++i) {
     indentStr += " ";
   }
-  if (_asString.size() == 0) {
```

Comment:
If we want to keep this caching we could also remember the `indent` that was used, right?